pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -584,6 +585,36 @@ def get_steinhardt_parameter(self, l, cutoff_radius=np.inf):
             for m in np.arange(-l, l+1)
         ], axis=0))
 
+    @staticmethod
+    def _get_all_possible_pairs(l):
```

Comment:
This function works maybe up to `l=10` but after that it takes too much memory and fails. If there's someone who can come up with a nice idea let me know. It in principle does what's described on [this page](https://stackoverflow.com/questions/5360220/how-to-split-a-list-into-pairs-in-all-possible-ways).